src/common/DisabledButtonInfo.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactNode, useEffect } from 'react'
+import Tooltip, { TooltipProps } from '@mui/material/Tooltip'
+
+function DisabledButtonInfo({
+  conditions,
+  placement,
+  children,
+}: {
+  conditions: string[]
+  placement?: TooltipProps['placement']
+  children: ReactNode
+}) {
+  const [titleText, setTitleText] = React.useState<string>('')
+
+  useEffect(() => {
+    if (conditions !== undefined) {
```

Comment:
By using this logic we're effectively changing the API of MUI's `Tooltip`. Would it not be better to stick with simply extending `TooltipProps` and managing the `title` outside of this component as we would with a regular tooltip?